backend/eventloop/shared_resource_loop/sharedresourceloop_test.go

```diff
@@ -154,6 +154,8 @@ var _ = Describe("SharedResourceEventLoop Test", func() {
 			Expect(err).To(BeNil())
 			Expect(usrNew).NotTo(BeNil())
 			Expect(isNewUser).To(BeFalse())
+			Expect(usrNew.Created_on.After(time.Now().Add(time.Minute*-5))).To(BeTrue(), "Created on should be within the last 5 minutes")
+			usrNew.Created_on = usrOld.Created_on
```

Comment:
Since the user was created previously, shouldn't `usrNew` and `userOld` be identical?  In which case, their  `Created_on` timestamp should be (must be) exactly the same, and we should test for that i.e. this line setting `usrNew.Created_on = usrOld.Created_on` should be removed (and a comment should be added explaining this).

It might also be clearer if the check that Created_on should be within the last five minutes was done on `usrOld` at the time `usrOld` is created.

Reply:
created_on for usrOld is  `Created_on: 2023-03-10T15:39:48.851556+05:30` and for usrNew it is `2023-03-10T10:09:48.851556Z`